Tests failing at HEAD: TestRunExposeService

```
ok      github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl   1.457s
W0711 04:38:54.425007     971 cmd.go:149] -p POD is DEPRECATED and will be removed in a future version. Use exec POD instead.
W0711 04:38:54.425083     971 cmd.go:149] -p POD is DEPRECATED and will be removed in a future version. Use exec POD instead.
W0711 04:38:54.425089     971 cmd.go:149] -p POD is DEPRECATED and will be removed in a future version. Use exec POD instead.
W0711 04:38:54.425125     971 cmd.go:149] -p POD is DEPRECATED and will be removed in a future version. Use exec POD instead.
W0711 04:38:54.425378     971 cmd.go:149] -p POD is DEPRECATED and will be removed in a future version. Use exec POD instead.
--- FAIL: TestRunExposeService (0.00s)
        expose_test.go:173: expose-external-service: unexpected output: &api.Service{TypeMeta:api.TypeMeta{Kind:"", APIVersion:""}, ObjectMeta:api.ObjectMeta{Name:"foo", GenerateName:"", Namespace:"", SelfLink:"", UID:"", ResourceVersion:"", Generation:0, CreationTimestamp:util.Time{Time:time.Time{sec:0, nsec:0, loc:(*time.Location)(nil)}}, DeletionTimestamp:(*util.Time)(nil), Labels:map[string]string{"svc":"test"}, Annotations:map[string]string(nil)}, Spec:api.ServiceSpec{Ports:[]api.ServicePort{api.ServicePort{Name:"default", Protocol:"UDP", Port:14, TargetPort:util.IntOrString{Kind:0, IntVal:14, StrVal:""}, NodePort:0}}, Selector:map[string]string{"func":"stream"}, ClusterIP:"", Type:"LoadBalancer", DeprecatedPublicIPs:[]string(nil), SessionAffinity:""}, Status:api.ServiceStatus{LoadBalancer:api.LoadBalancerStatus{Ingress:[]api.LoadBalancerIngress(nil)}}}
FAIL
FAIL    github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl/cmd       0.843s
```

--- a/pkg/kubectl/cmd/expose_test.go
+++ b/pkg/kubectl/cmd/expose_test.go
@@ -25,6 +25,7 @@ import (
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/api"
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/client"
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/runtime"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/util"
 )
 
 func TestRunExposeService(t *testing.T) {
@@ -126,17 +127,17 @@ func TestRunExposeService(t *testing.T) {
 				Spec: api.ServiceSpec{
 					Ports: []api.ServicePort{
 						{
-							Name:     "default",
-							Protocol: api.Protocol("UDP"),
-							Port:     14,
+							Name:       "default",
+							Protocol:   api.Protocol("UDP"),
+							Port:       14,
+							TargetPort: util.NewIntOrStringFromInt(14),
 						},
 					},
 					Selector: map[string]string{"func": "stream"},
 					Type:     api.ServiceTypeLoadBalancer,
 				},
 			},
-			expected: "you will also need to explicitly open a firewall",
-			status:   200,
+			status: 200,
 		},
 	}
 

